configuration/outputs.md

```diff
@@ -361,6 +361,109 @@ If enabled, the SignalFx integration will stream analytics and insights to an Si
 
 ## Trigger Destinations
 
+### **Notify Content**
```

Comment:
move this below the Slack section and also use a smaller title font. `####`

Reply:
makes sense, moved it to below slack section